franklin/builder/tests.py

```diff
@@ -78,25 +80,50 @@ def test_get_deployable_event_tag(self):
         else:
             self.fail(github_event.errors)
 
-    def test_building_env(self):
+    @mock.patch('core.helpers.requests.post')
+    def test_building_env(self, mock_post):
         """ Tests the model method that calls franklin-builder when an
         environment is ready to be deployed for a branch.
         """
+        # Mock the POST request to builder
+        response = mock.Mock()
+        response.status_code = 200
+        mock_post.return_value = response
+
+        self.assertEqual(self.env.status, Environment.REGISTERED)
```

Comment:
This test had the potential to call an actual api previously. Am now mocking the requests library endpoint to return a success. We will want to use this kind of mocking more in the future.